nextmv/nextmv/cloud/manifest.py

```diff
@@ -520,12 +609,45 @@ class ManifestOptions(BaseModel):
 
     strict: Optional[bool] = False
     """If strict is set to `True`, only the listed options will be allowed."""
+    validation: Optional[ManifestValidation] = None
+    """Optional validation rules for all options."""
     items: Optional[list[ManifestOption]] = None
-    """The actual list of options for the decision model.
+    """The actual list of options for the decision model."""
```

Comment:
```suggestion
    """The actual list of options for the decision model.

    An option is a parameter that configures the decision model.
    """
```

If the removal of the "An option is a parameter that configures the decision model." was intentional, ignore me. :blush: